src/AppInstallerCLICore/Workflows/ShowFlow.cpp

```diff
@@ -175,36 +154,18 @@ namespace AppInstaller::CLI::Workflow
         {
             Manifest::InstallerTypeEnum effectiveInstallerType = installer->EffectiveInstallerType();
             Manifest::InstallerTypeEnum baseInstallerType = installer->BaseInstallerType;
-            if (effectiveInstallerType == baseInstallerType)
-            {
-                info << "  "_liv << Execution::ManifestInfoEmphasis << Resource::String::ShowLabelInstallerType << ' ' << Manifest::InstallerTypeToString(effectiveInstallerType) << std::endl;
-            }
-            else
-            {
-                info << "  "_liv << Execution::ManifestInfoEmphasis << Resource::String::ShowLabelInstallerType << ' ' << Manifest::InstallerTypeToString(effectiveInstallerType) << " (" <<
-                    Manifest::InstallerTypeToString(baseInstallerType) << ')' << std::endl;
-            }
-
-            if (!installer->Locale.empty())
-            {
-                info << "  "_liv << Execution::ManifestInfoEmphasis << Resource::String::ShowLabelInstallerLocale << ' ' << installer->Locale << std::endl;
-            }
-            if (!installer->Url.empty())
-            {
-                info << "  "_liv << Execution::ManifestInfoEmphasis << Resource::String::ShowLabelInstallerUrl << ' ' << installer->Url << std::endl;
-            }
-            if (!installer->Sha256.empty())
-            {
-                info << "  "_liv << Execution::ManifestInfoEmphasis << Resource::String::ShowLabelInstallerSha256 << ' ' << Utility::SHA256::ConvertToString(installer->Sha256) << std::endl;
-            }
-            if (!installer->ProductId.empty())
-            {
-                info << "  "_liv << Execution::ManifestInfoEmphasis << Resource::String::ShowLabelInstallerProductId << ' ' << installer->ProductId << std::endl;
-            }
-            if (!installer->ReleaseDate.empty())
+            std::stringstream shownInstallerType;

```

Comment:
I'm not sure if a stringstream is necessary here, can it be just the std::string